test/e2e/deploy/deploy.go

```diff
@@ -90,8 +98,14 @@ func WaitForPMEMDriver(c *Cluster, name, namespace string, testing bool) (metric
 	defer ticker.Stop()
 	info := time.NewTicker(time.Minute)
 	defer info.Stop()
-	deadline, cancel := context.WithTimeout(context.Background(), framework.TestContext.SystemDaemonsetStartupTimeout)
+	deadline, cancel := context.WithTimeout(context.Background(), 10*time.Minute)
 	defer cancel()
+
+	if waitForPMEMDriverTimedOut {
+		// Abort early.
+		skipper.Skipf("installing PMEM-CSI driver during previous test was too slow")
+	}
```

Comment:
I think this is indeed useful: in https://cloudnative-k8sci.southcentralus.cloudapp.azure.com/blue/organizations/jenkins/pmem-csi/detail/version-skew-2/19/tests the same driver install error would have happened for all skew tests, so instead of repeating that over and over, testing runs into that issue once and then skips the rest of the testing.

We just have to remember that "only one failed test" no longer means "all other tests were okay".

Reply:
FWIW, that issue is known and a result of rebasing on top of the metrics code: now the version skew testing must switch between http and https depending on the version.

Reply:
Yes, Let's keep this. It's useful for quick fail.